lib/app/views/account/show.erb

```diff
@@ -47,26 +47,7 @@
           <h2>Exercises</h2>
           <h4>Current</h4>
           <% if profile.has_current_submissions? %>
-            <table class="table table-bordered table-striped">
-              <thead>
-                <tr>
-                  <th>Exercise</th>
-                  <th>Language</th>
-                  <th>Current Iteration</th>
-                  <th>Nits</th>
-                </tr>
-              </thead>
-              <tbody>
-              <% profile.current.each do |submission| %>
-                <tr>
-                  <td><%= profile.submission_link(submission) %></td>
-                  <td><%= Language.of(submission.track_id) %></td>
```

Comment:
Yeah, that's a known bug. The `Language.of` returns the name of the language with correct case etc, whereas the submission just knows the track id (all lowercase, maybe with a hyphen). But `Language.of` relies on a hard-coded list in the `Exercism::Config`, and we keep adding more languages. https://github.com/exercism/exercism.io/issues/2356